tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+	"compilerOptions": {
+		"allowJs": true,
+		"module": "commonjs",
+		"target": "es5",
+		"noImplicitAny": true,
+		"jsx": "react",
+		"outDir": "./dist/",
+		"noFallthroughCasesInSwitch": true,
+		"noImplicitReturns": true,
+		"noImplicitThis": true,
+		"noUnusedLocals": true,
+		"noUnusedParameters": true,
+		"pretty": true,
+		"removeComments": true,
+		"strictNullChecks": true
+	},
+	"include": [
+		"./app/**/*"
+	],
+	"exclude": [
+		"node_modules"
+	]
```

Comment:
On second thought, I don't get the impression that these are mutually exclusive: https://www.typescriptlang.org/docs/handbook/tsconfig-json.html#details It looks like adding something in `include` doesn't necessary mean only that folder tree is compiled. So we might have to add the `exclude` directive as well. 🤔